src/k8s-configuration/azext_k8s_configuration/_params.py

```diff
@@ -118,6 +118,15 @@ def load_arguments(self, _):
                    options_list=['--yes', '-y'],
                    help='Do not prompt for confirmation')
 
+    with self.argument_context('k8s-configuration flux deployed-object show') as c:
+        c.argument('object_name',
+                   help='Name of the object deployed by the configuration on the cluster.')
+        c.argument('object_namespace',
+                   help='Namespace of the object deployed by the configuration on the cluster.')
+        c.argument('object_kind',
+                   arg_type=get_enum_type(['GitRepository', 'Bucket', 'HelmRepository', 'HelmChart', 'HelmRelease', 'Kustomization']),
```

Comment:
May I ask could we get those enumeration values from the Python SDK?

Reply:
We do not currently have these types defined in the swagger but will define them at the next iteration